Sources/ComposableArchitecture/SharedState/PersistenceKey.swift

```diff
@@ -7,9 +7,9 @@
 ///
 /// See the article <doc:SharingState> for more information, in particular the
 /// <doc:SharingState#Custom-persistence> section.
-public protocol PersistenceReaderKey<Value> {
+public protocol PersistenceReaderKey<Value>: Sendable {
   /// A type that can be loaded or subscribed to in an external system.
-  associatedtype Value
+  associatedtype Value: Sendable
```

Comment:
`Value` is captured in a variety of sendable contexts, such as in the subscription endpoint for persistence strategies. So I think this does really need to be full blown `Sendable`.